Tools/PrivateRepoLib.ps1

```diff
@@ -0,0 +1,113 @@
+
+
+Function New-WinGetManifest
+{
+    <#
+    .SYNOPSIS
+    Submits Manifest files to the Azure Private REpository
+    
+    .DESCRIPTION
+    By running this function with the required inputs, it will connect to the Azure Tennant that hosts the Windows Package Manager Private Repository, then collects the required URL for Manifest submission before retrieving the contents of the Manifest JSON to submit.
+    
+    .PARAMETER PrivateRepoName
+    Name of the Windows Package Manager Private repository. Can be identified by running: "winget source list" and using the Repository Name
+    
+    .PARAMETER AzureFunctionName
+    Name of the Azure Function that hosts the Private Repository
+    
+    .PARAMETER ManifestFilePath
+    Path to the JSON manifest file that will be uploaded to the Private Repository
```

Comment:
What about multifiled manifests? we should also take a directory and use the client manifest parser to grok the combined app information.

Reply:
I can add a folder option but I might require some help on handling "multified manifests that can make use of the client manifest parser to grok the combined app information", as I'm not sure I understand.

Reply:
@jamespik